src/wp-admin/network/user-new.php

```diff
@@ -100,7 +100,8 @@
 $title       = __( 'Add New User' );
 $parent_file = 'users.php';
 
-require_once ABSPATH . 'wp-admin/admin-header.php'; ?>
```

Comment:
This doesn't look like the related change to the current Trac Ticket.

Reply:
Thank you, I've reverted the changes.